docs/network-flow-visibility.md

```diff
@@ -109,9 +109,14 @@ parameters have to be set in the Antrea Agent ConfigMap:
 
 Please note that the default value for `flowCollectorAddr` is `"flow-aggregator.flow-aggregator.svc:4739:tcp"`,
 which uses the DNS name of the Flow Aggregator Service, if the Service is deployed
-with the Name and Namespace set to `flow-aggregator`. If you deploy the Flow Aggregator
-Service with a different Name and Namespace, then either use the appropriate DNS
-name or the Cluster IP of the Service. Please note that the default values for
+with the Name and Namespace set to `flow-aggregator`. For Antrea Agent running on 
+a Windows node, the user is required to change the default value of `HOST` in `flowCollectorAddr` 
+from DNS name to the Cluster IP of Flow Aggregator service. The reason is because 
+of the DNS resolution issue in the current Windows support of Kubernetes. In addition, 
```

Comment:
This is too vague. Please provide more context & point to a Github issue. The issue is specific to the userspace kube-proxy implementation on Windows, which Antrea currently depends on.

Reply:
Sure, I didn't find an exisitng issue specific to the DNS resolve problem of kube-proxy on Windows, are these two issue good to cited here: https://github.com/vmware-tanzu/antrea/issues/1935, https://github.com/vmware-tanzu/antrea/issues/1954?